packages/stellar-axelar-std-derive/src/lib.rs

```diff
@@ -280,7 +280,7 @@ fn ensure_no_args(attr: &Attribute) -> syn::Result<&Path> {
 /// }
 /// }
 /// ```
-#[proc_macro_derive(IntoEvent, attributes(data, event_name))]
+#[proc_macro_derive(IntoEvent, attributes(data, event_name, singleton_data))]
```

Comment:
```suggestion
#[proc_macro_derive(IntoEvent, attributes(event_name, datum, data))]
```
A bit cryptic if someone doesn't know but I like @nbayindirli's `datum` proposal